articles/how-to-use-local-storage-with-JavaScript/index.md

```diff
@@ -0,0 +1,284 @@
+Local Storage allows developers to store and retrieve data in the browser. The data stored in the local storage doesn't expire. This means the data will persist even if the tab or the browser window is closed.
+
+### Prerequisites
+You must have a basic understanding of javascript. You also need a code editor and browser to test the project. In this tutorial, we will be using Visual Studio Code and Chrome.
+
+### What is Local Storage
+Local Storage is a web storage that stores data for a long time. This could be a day, a week, or even a year. This depends upon the developer's preference. It is important to note that local storage only stores strings. So, if you wish to store objects, lists, or arrays, you must convert them into a string using `JSON.stringify().`
+
+### When to use Local Storage
+You should only use local storage when storing insensitive information. This is because third-party individuals can easily access the information. Local storage can help in storing temporary data before it is pushed to the server. It is important to clear the local storage once this operation is completed.
+
+### Limitations
+The major limitations of local storage are:
+
+- Insecure data.
+
+- Synchronous operations.
+
+- Limited storage capacity.
+
+### Main methods in Local Storage
+The primary methods when using Local Storage are `key()`, `setItem()`, `removeItem()`, `getItem()`, and `clear()`.
+
+#### key()
+This method is used to retrieve a value/string from a specific location. The index can be passed into the `key()` function as a parameter.
+
+```JavaScript
+var answer = localStorage.key(1);
+// this statement will retrieve the value of the second item in the localStorage.
+```
+
+The `key()` can also be used in a loop statement to retrieve all the items in the local storage.
+
+#### setItem()
+This function is used to store items in the local storage. An example of this function is shown below.
+
+```JavaScript
+Window.localStorage.setItem("grade","One");
```

Comment:
`Window` is wrong. It's `window`. JS is case sensitive. Please fix this everywhere in the article.